swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
I'd suggest flipping the sense of the condition around to handle the special case (that something is falsy) quickly and get out, rather than having that dangling else at the end. This arrangement is referred to as a guard clause or a guard condition. Notice that it lets us move the main focus of the function (removing the item) out of an indented block, letting it stand out more clearly.
```python
def remove(self, item):
    if item not in self.inventory:
        return False
    self.inventory.remove(item)
    return item
```